python/rzpipe/open_base.py

```diff
@@ -59,13 +59,11 @@ def json2obj(data):
 
 
 def get_rizin_path():
-    try:
-        which = shutil.which
-    except AttributeError:
-        # In python 2 doesn't exist which function
-        from distutils.spawn import find_executable
-
-        which = find_executable
+    """
+    Python 2/3 agnostic helper to get the path to the rizin binary.
```

Comment:
Since we don't support Python 2 anymore, probably better to edit this description.

Reply:
Nice, ok

Reply:
btw. I also threw away legacy python 3.4-3.5 asyncio stuff.. Also explicit loop passing is actually deprecated

Reply:
Right, most modern distributions provide python 3.6 at least: https://repology.org/project/python/versions

Reply:
Ok perfect.

Reply:
You forgot to remove the `Python 2` mention.

Reply:
yeah, will remove.. I am not finished tho with the tests and stuff. kinda busy these days

Reply:
there are also some fixup commits which I will squash later, I don't like blunder commits